<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgCompareTool</class>
 <widget class="QDialog" name="ImgCompareTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Comparison Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labLeftCompareImg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleName">
        <string>imgPreview</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labRightCompareImg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleName">
        <string>imgPreview</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labResult">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="btnLoadLeft">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCompare">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Compare</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoadRight">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnLoadLeft</sender>
   <signal>clicked()</signal>
   <receiver>ImgCompareTool</receiver>
   <slot>onLoadLeftImg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoadRight</sender>
   <signal>clicked()</signal>
   <receiver>ImgCompareTool</receiver>
   <slot>onLoadRightImg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCompare</sender>
   <signal>clicked()</signal>
   <receiver>ImgCompareTool</receiver>
   <slot>onCompare()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLoadLeftImg()</slot>
  <slot>onLoadRightImg()</slot>
  <slot>onCompare()</slot>
 </slots>
</ui>
